src/route.c

```diff
@@ -587,7 +587,7 @@ static void fragment_ipv4_packet(node_t *dest, vpn_packet_t *packet, length_t et
 	todo = ntohs(ip.ip_len) - ip_size;
 
 	if(ether_size + ip_size + todo != packet->len) {
-		logger(DEBUG_TRAFFIC, LOG_WARNING, "Length of packet (%d) doesn't match length in IPv4 header (%d)", packet->len, (int)(ether_size + ip_size + todo));
+		logger(DEBUG_TRAFFIC, LOG_WARNING, "Length of packet (%d) doesn't match length in IPv4 header (%lu)", packet->len, ether_size + ip_size + todo);
```

Comment:
Should be `%zu` instead of `%lu`, since it's a `size_t`?

Reply:
Makes sense, thanks. To be honest, I am more used to linters for more high level languages, which do not produce so many broken auto"fixes". Here you cannot really rely on what the linter recommends. I'll double check it in the future.

The static analysis job here failed to checkout code (GitHub Actions are not particularly reliable), so here's a second [test run](https://github.com/hg/tinc/actions/runs/1089778614).